### PR TITLE
ebusd sensor component documentation

### PR DESCRIPTION
**Pull request by CrazYoshi**
Integration between ebusd (https://github.com/john30/ebusd), daemon for communication with eBUS heating systems, and homeassistant using sensor component.

### CODE REVIEW

**Comment by CrazYoshi** [comment]
I don't understand why I get this error during deploy validation `failed during stage 'deploying site': Invalid filename 'getting-started/basic/#remote-access.html'. Deployed filenames cannot contain # or ? characters`

File: source/_components/sensor.ebusd.markdown [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+comments: false
+sharing: true
+footer: true
+logo: ebusd.png
+ha_category: Sensor
+ha_iot_class: "Local Push"
+ha_release: 0.79
+---
+
+Integration between ebusd (https://github.com/john30/ebusd), daemon for communication with eBUS heating systems, and homeassistant using sensor component.
```
Comment (WofWca) [inline source/_components/sensor.ebusd.markdown]: What about `[ebusd](https://github.com/john30/ebusd)`?

File: source/_components/sensor.ebusd.markdown [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+title: "eBusd"
```
Comment (WofWca) [inline source/_components/sensor.ebusd.markdown]: Do you think this is written in the right case? It's "ebusd" nearly everywhere.

**Review by frenck — CHANGES_REQUESTED** [review]
- Please restore the PR template in the opening post.
- Link this issue to the parent PR in the main repo.
- Rebase this PR onto `origin/next` to resolve the build issues.

**Comment by frenck** [comment]
Not sure why you closed this PR @CrazYoshi?

**Comment by WofWca** [comment]
@frenck He replaced it with #6433.

**Comment by frenck** [comment]
Thx @WofWca

**Pull request closed without merging**